tests/test_cases/getcwd.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+#include <unistd.h>
+
+#define BUFSIZE 64
```

Comment:
I think it would be better to opt for a larger bufsize ~2048 in case we want to do tests of getcwd  within deeply nested directories? Not a big deal at all though,